src/pages/registration/DescriptionPanel.tsx

```diff
@@ -84,20 +84,53 @@ export const DescriptionPanel = () => {
           )}
         </Field>
       </Box>
-      <Field name={DescriptionFieldNames.Abstract}>
-        {({ field }: FieldProps<string>) => (
-          <TextField
-            {...field}
-            value={field.value ?? ''}
-            data-testid={dataTestId.registrationWizard.description.abstractField}
-            variant="filled"
-            fullWidth
-            multiline
-            rows="4"
-            label={t('registration.description.abstract')}
-          />
-        )}
-      </Field>
+      <Box
+        sx={{
+          display: 'grid',
```

Comment:
tenkte at en flex-row var mer naturlig her, da det bare er to elemente. Men ser vi har gjort det slik over her også så kanskje det er en grunn til det :p